app/views/commontator/comments/_show.html.erb

```diff
@@ -67,27 +67,39 @@
-              id: "commontator-comment-#{comment.id}-edit",
-              class: "fas fa-edit fa-lg edit-comment-icon #{ creator.first_name.blank? && creator.last_name.blank? ? '' : 'margin-left-1' }",
-              remote: true
-          ) if comment.can_be_edited_by?(user)
-        %>
-
-        <% if comment.can_be_deleted_by?(user) %>
-          <% is_deleted = comment.is_deleted? %>
-          <% del_string = is_deleted ? 'undelete' : 'delete' %>
-          <%= link_to '',
-                      is_deleted ? commontator.undelete_comment_path(comment) : commontator.delete_comment_path(comment),
-                      data: is_deleted ? { confirm: "Are you sure you want to restore this #{comment.parent.nil? ? 'comment' : 'reply'}?"} : { confirm: "Are you sure you want to delete this #{comment.parent.nil? ? 'comment' : 'reply'}?" },
-                      method: :put,
-                      id: "commontator-comment-#{comment.id}-#{del_string}",
-                      class: is_deleted ? "fas fa-trash-restore-alt fa-lg margin-left-1" : "fas fa-trash-alt fa-lg delete-comment-icon margin-left-1",
-                      remote: true %>
+        <% if !@practice.retired %>
+          <%=
+            link_to(
+                '',
+                commontator.edit_comment_path(comment),
+                id: "commontator-comment-#{comment.id}-edit",
+                class: "fas fa-edit fa-lg edit-comment-icon #{ creator.first_name.blank? && creator.last_name.blank? ? '' : 'margin-left-1' }",
+                remote: true
+            ) if comment.can_be_edited_by?(user)
+          %>
+         <% else %>
+          <a class="fas fa-edit fa-lg edit-comment-icon margin-left-1"></a>
         <% end %>
+
+          <% if comment.can_be_deleted_by?(user) %>
+            <% is_deleted = comment.is_deleted? %>
+            <% del_string = is_deleted ? 'undelete' : 'delete' %>
+            <% if !@practice.retired %>
+                <%= link_to  '',
+                             is_deleted ? commontator.undelete_comment_path(comment) : commontator.delete_comment_path(comment),
+                             data: is_deleted ? { confirm: "Are you sure you want to restore this #{comment.parent.nil? ? 'comment' : 'reply'}?"} : { confirm: "Are you sure you want to delete this #{comment.parent.nil? ? 'comment' : 'reply'}?" },
+                             method: :put,
+                             id: "commontator-comment-#{comment.id}-#{del_string}",
+                             class: is_deleted ? "fas fa-trash-restore-alt fa-lg margin-left-1" : "fas fa-trash-alt fa-lg delete-comment-icon margin-left-1",
+                             remote: true %>
+            <% else %>
+                  <% if is_deleted %>
+                        <a class="fas fa-trash-restore-alt fa-lg margin-left-1"></a>
+                  <% else %>
+                    <a class="fas fa-trash-alt fa-lg delete-comment-icon margin-left-1"></a>
```

Comment:
Can we make this one element with a ternary statement and include classes accordingly?

Reply:
Done.